### PR TITLE
added system container instructions

### PR DESCRIPTION
**Pull request by jtligon**
Once Centos and Fedora containers are ready, these instructions will help a user install via system containers.

Waiting for final word from @ashcrow and @giuseppe

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -23,6 +23,21 @@ Join #cri-o on IRC (freenode)
...
+
+* **Fedora and Fedora Atomic Host**: Available on 26+ and Rawhide
+
+        atomic install --system-package=no -n cri-o --storage ostree
+docker.io/gscrivano/cri-o-fedora
+
+* **Centos and Centos Atomic Host**: Available 7.4+
+
+        atomic install --system-package=no -n cri-o --storage ostree
+docker.io/gscrivano/cri-o-centos
```
Comment (giuseppe) [inline index.md]: we could probably already use the CentOS registry location here.

We can update the Fedora registry once we have it.

Comment (ashcrow) [reply index.md]: @giuseppe Can you paste the CentOS location here?

Comment (giuseppe) [reply index.md]: here it is: registry.centos.org/projectatomic/cri-o:latest

Comment (ashcrow) [reply index.md]: @jtligon 

```
atomic install --system-package=no -n cri-o --storage ostree registry.centos.org/projectatomic/cri-o:latest
```

**Pull request merged**